app/assets/javascripts/hyrax/collections.js

```diff
@@ -245,9 +245,9 @@ Blacklight.onLoad(function () {
       // Get all inputs in the table
       .find('td:first input[type=checkbox]')
       // Filter to those that are checked
-      .filter((i, checkbox) => checkbox.checked)
+      .filter(function(i, checkbox) { return checkbox.checked });
```

Comment:
To be safe, put an ending semicolon on the `return` statement?  `.filter(function(i, checkbox) { return checkbox.checked; });`

Reply:
Ah, good catch. :)